src/steamship/base/base.py

```diff
@@ -1,9 +1,27 @@
+import re
 from typing import TypeVar
 
+import inflection
+from pydantic import BaseModel
 from pydantic.generics import GenericModel
 
 T = TypeVar("T")  # Declare type variable
 
 
-class IResponse(GenericModel):
```

Comment:
We don't need the interface anymore 🎉 Let's nuke these circular dependencies ☢️😈